### PR TITLE
add missing R5 source mapping for ST60 series

### PR DESCRIPTION
**Pull request by lealoureiro**
I was trying the current code with Home-Assistant and i notice that some source mapping was till missing.

I also started ignoring the `0x00` byte coming from ST60 as discussed before.

I've also replaced the way the increment and decrement volume commands are sent, since the RC commands only support navigate up and navigate down, which changes the volume when device is in home screen, if the device is in another screen/menu does not work.

Using native commands is more clean and more robust.

### CODE REVIEW

File: src/arcam/fmj/__init__.py [hunk 1]
```diff
@@ -696,6 +700,22 @@ class DecodeModeMCH(IntOrTypeEnum):
...
+        SourceCodes.NET: bytes([21, 92])
+    },
+    (ApiModel.APIST_SERIES, 2): {
+        SourceCodes.DIG1: bytes([21, 94]),
+        SourceCodes.DIG2: bytes([21, 98]),
+        SourceCodes.DIG3: bytes([21, 27]),
+        SourceCodes.DIG4: bytes([21, 27]),
+        SourceCodes.USB: bytes([21, 93]),
+        SourceCodes.NET: bytes([21, 92])
+    },
```
Comment (elupus) [inline src/arcam/fmj/__init__.py:718]: This looks wrong. If they device doesn't support multiple zones, it should at least not list sources for zone 2.

Comment (lealoureiro) [reply src/arcam/fmj/__init__.py:718]: I'm not really sure if device supports multiple zones. Do you know if I can spot that from docs? 
I will have a look again and remove it. Also I saw in HA that a 2nd device with name Zone 2 is created but is not usable I think.

Comment (elupus) [reply src/arcam/fmj/__init__.py:718]: If it does support it it, it should have different rc codes for the zones.

Comment (lealoureiro) [reply src/arcam/fmj/__init__.py:718]: I see this in document

```quote
Zone numbers
The following zone numbers are defined:
• 0x01 – Zone number 1. (Zone 1 is the master zone. Commands that appear zone-less refer to the master zone)
• 0x02 – Zone number 2
```

Comment (elupus) [reply src/arcam/fmj/__init__.py:718]: The if there are no distinct RC codes, it wont be able to set the source directly. Ie changing source on zone 2 will affect zone one. Normally there are distinct rc codes per zone.

Comment (lealoureiro) [reply src/arcam/fmj/__init__.py:718]: I'm a bit confused, sorry, so you suggest to remove the mapping for Zone 2?

Comment (elupus) [reply src/arcam/fmj/__init__.py:718]: Yes

Comment (lealoureiro) [reply src/arcam/fmj/__init__.py:716]: removed as suggested!

**Review by elupus — APPROVED** [review]
Looks good! Thanx

**Pull request merged**